src/Uno.UI.BindingHelper.Android/Uno/UI/UnoTwoDScrollView.java

```diff
@@ -1191,20 +1191,6 @@ public void requestLayout() {
 		super.requestLayout();
 	}
 
-	@Override
-	protected void onLayout(boolean changed, int l, int t, int r, int b) {
-		super.onLayout(changed, l, t, r, b);
-		mIsLayoutDirty = false;
-		// Give a child focus if it needs it
-		if (mChildToScrollTo != null && isViewDescendantOf(mChildToScrollTo, this)) {
-			scrollToChild(mChildToScrollTo);
-		}
-		mChildToScrollTo = null;
-
-		// Calling this with the present values causes it to re-clam them
-		scrollTo(getScrollX(), getScrollY());
-	}
-
```

Comment:
@jeromelaban @davidjohnoliver This code is not called anymore.  Do you think it's doing something important?

Reply:
Typically when the native code tries to scroll a child into view it's something we don't want, so this might actually be an improvement.

Reply:
Never mind... I'll revert this change, I found a bette fix.